app/cdap/components/RecipeDetails/DrawerHeader/index.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { grey } from '@material-ui/core/colors';
+import React from 'react';
+import styled from 'styled-components';
+import CloseIcon from '@material-ui/icons/Close';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import EditIcon from '@material-ui/icons/Edit';
+import SaveAltIcon from '@material-ui/icons/SaveAlt';
+import { Box, IconButton, Typography } from '@material-ui/core';
+import T from 'i18n-react';
+import { UnderlineIcon } from 'components/RecipeDetails/IconStore/UnderlineSVG';
+
+interface IDrawerHeaderProps {
+  onCloseDetail: () => void;
+}
+
+const MainHeadWrapper = styled(Box)`
+  margin-top: 20px;
+`;
+
+const HeadWrapper = styled(Box)`
```

Comment:
check if we same component in the existing or new code wr.t this pr
and update it accordingly